examples/plot_decoding_tutorial.py

```diff
@@ -195,18 +197,13 @@
 # ...................................
 #
 # Scikit-learn has tools to perform cross-validation easier:
-from sklearn.cross_validation import cross_val_score
+from sklearn.model_selection import cross_val_score
 cv_score = cross_val_score(svc, fmri_masked, conditions)
 print(cv_score)
 
 ###########################################################################
 # Note that we can speed things up to use all the CPUs of our computer
 # with the n_jobs parameter.
-#
-# By default, cross_val_score uses a 3-fold KFold. We can control this by
-# passing the "cv" object, here a 5-fold:
-cv_score = cross_val_score(svc, fmri_masked, conditions, cv=cv)
-print(cv_score)
```

Comment:
It has not been removed, a little rearranged.
@GaelVaroquaux You should take a look at this section `Line Number 177 to 232`, I have reworded the comments & moved a few lines of code and comments to reflect the new structure wrought by `model_selection`. I think I have done a reasonable job, but it would be best if you took a look at the entire section.